homeassistant/components/reolink/host.py

```diff
@@ -131,6 +135,27 @@ async def async_init(self) -> None:
 
         await self.subscribe()
 
+        _LOGGER.debug("Checking webhook '%s' is reachable", self.webhook_id)
+        session = async_get_clientsession(self._hass, verify_ssl=False)
+        try:
+            await session.post(self._webhook_url, data=CHECK_WEBHOOK_TEXT)
+            await asyncio.wait_for(self._webhook_reachable.wait(), timeout=5)
```

Comment:
This is not a valid test.
You cannot test if a camera can reach a Home Assistant webhook URL from Home Assistant itself.

The only valid/correct way to test this is from the device, not Home Assistant.

Reply:
@frenck you are right, from further inverstigation I found out the reolink camera's publish there initial motion/AI state to the ONVIF WS base notification webhook upon subscription. Therefore I can just wait for that initial push to come in to confirm the webhook URL is correct and the reolink camera can reach it.

On my E1 Zoom and RLN8 NVR it takes less than a second from the moment I subscibe to the webhook beeing called by the camera publishing its initial state.

I have adjusted this PR accordingly.